ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN curl --silent --location  https://rpm.nodesource.com/setup_lts.x | bash - && \
```

Comment:
`https://rpm.nodesource.com/setup_lts.x` to `https://rpm.nodesource.com/setup_16.x` to continue to test your PR. 
after it is fixed, you could change it back.